rules.bzl

```diff
@@ -16,6 +16,15 @@ def _buildbuddy_toolchain_impl(rctx):
     else:
         default_platform = "platform_linux"
 
+    default_container_image = _container_image_prop(rctx.attr.container_image)
+    if default_container_image == None or default_container_image == "":
+        if rctx.os.arch == "aarch64":
+            default_container_image = _container_image_prop(UBUNTU22_04_IMAGE)
+        else:
+            default_container_image = _container_image_prop(UBUNTU20_04_IMAGE)
```

Comment:
The 22.04 image is multi-platform at this point right? If so I think it's fine to use UBUNTU22_04_IMAGE as the default regardless of arch. People depend on a fixed commit from this toolchain repo so this shouldn't break any existing usages

Reply:
done